swift/Sources/FlatBuffers/FlatbuffersErrors.swift

```diff
@@ -57,6 +55,59 @@ public enum FlatbuffersErrors: Error, Equatable {
     lhs: FlatbuffersErrors,
     rhs: FlatbuffersErrors) -> Bool
   {
-    lhs.localizedDescription == rhs.localizedDescription
+      switch lhs {
+      case .unknownUnionCase:
+          return rhs == .unknownUnionCase
+      case .exceedsMaxSizeAllowed:
+          return rhs == .exceedsMaxSizeAllowed
+      case .missAlignedPointer(position: let position, type: let type):
+          if case .missAlignedPointer(let rhsPosition, let rhsType) = rhs {
+              return position == rhsPosition && type == rhsType
+          } else {
+              return false
+          }
+      case .outOfBounds(position: let position, end: let end):
+          if case .outOfBounds(let rhsPosition, let rhsEnd) = rhs {
+              return position == rhsPosition && end == rhsEnd
+          } else {
+              return false
+          }
+      case .signedOffsetOutOfBounds(offset: let offset, position: let position):
+          if case .signedOffsetOutOfBounds(let rhsOffset, let rhsPosition) = rhs {
+              return offset == rhsOffset && rhsPosition == position
+          } else {
+              return false
+          }
+      case .requiredFieldDoesntExist(position: let position, name: let name):
+          if case .requiredFieldDoesntExist(let rhsPosition, let rhsName) = rhs {
+              return rhsPosition == position && rhsName == name
+          } else {
+              return false
+          }
+      case .missingNullTerminator(position: let position, str: let str):
+          if case .missingNullTerminator(let rhsPosition, let rhsStr) = rhs {
+              return rhsPosition == position && rhsStr == str
```

Comment:
What can be done here is moving this into the test module, I kinda forgot to move it